rust/noosphere-name-system/examples/pinned.toml

```diff
@@ -0,0 +1,3 @@
+[test_sphere]
```

Comment:
This TOML with pinned keys is purely for dev/testing. Maybe some existing infra in `noosphere-gateway`, a sqlite table, or simple files to track "pinned content" (using IPFS parlance) while running the node. Any insight here @cdata ?

The CLI was originally an example, hence this toml file being here. Probably should move it back to being an example. Depends on what the final interface looks like with the gateway. Sort of inspired by (mostly now deprecated) kubo CLI for interfacing with the DHT via low level commands for debugging/customization.

Reply:
Yah, use [`SphereDb`](https://github.com/subconsciousnetwork/noosphere/blob/88b08e3dc6e9df8386ed109b81bb38c8f06ceb4f/rust/noosphere-storage/src/db.rs#L142-L159) or any [`Store`](https://github.com/subconsciousnetwork/noosphere/blob/main/rust/noosphere-storage/src/interface.rs#L40-L55)-implementing construct. They implement [`KeyValueStore`](https://github.com/subconsciousnetwork/noosphere/blob/88b08e3dc6e9df8386ed109b81bb38c8f06ceb4f/rust/noosphere-storage/src/interface.rs#L185-L213), so you can `set_key`, `get_key` for configuration like this.

In the big picture, this data is going to come from the `Sphere` data structure itself, which means it will ultimately come from one of these stores (although it will be in a block, not a k/v entry).